src/main/java/mate/academy/hibernate/relations/lib/Dao.java

```diff
@@ -0,0 +1,8 @@
+package mate.academy.hibernate.relations.lib;
```

Comment:
Why do you need this lib package with all classes?

Reply:
decided to use injectors and then delete them but did not delete a folder.